phpseclib/File/X509.php

```diff
@@ -4631,7 +4631,7 @@ function _extractBER($str)
      * getOID('zzz') == 'zzz'
      *
      * @access public
-     * @return String
+     * @return string
```

Comment:
this wasn't caught by my original PR because it was merged into the codebase after I send the PR